### DISCUSSION TITLE
Are you still in need of beta testers?

### DISCUSSION THREAD

**Discussion by arooni** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#642] [Ideas]
I have a 2021 16" Macbook Pro M1 Max and would love to test future releases.  I am often using it in my home office but also in various other locations and the pro features are very compelling to me.

I am fairly active on github and have contributed a fair number of verified bugs to [koreader](https://github.com/koreader/koreader), as well as other repositories.

**Comment by MatthiasKerbl** [comment]
> I have a 2021 16" Macbook Pro M1 Max and would love to test future releases. I am often using it in my home office but also in various other locations and the pro features are very compelling to me.
> 
> I am fairly active on github and have contributed a fair number of verified bugs to [koreader](https://github.com/koreader/koreader), as well as other repositories.

Hi @arooni,

We really appreciate that you want to help us. Unfortunately, the beta phase is already long over.

Cheers,
Matthias
